src/js/modules/ThemeToggle.js

```diff
@@ -0,0 +1,28 @@
+
+export function setupThemeToggle() {
+    const themeToggle = document.querySelector('.theme-toggle');
+
+    const prefersDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
+    document.documentElement.setAttribute('data-theme', prefersDark ? 'dark' : 'light');
+    themeToggle.innerHTML = prefersDark
+        ? '<i class="fas fa-sun"></i>'
```

Comment:
use native HTML icon ( svg ) no extrenal library

Reply:
instead of using external library, Is using utf symbol fine?
[Sun Icon](https://www.w3schools.com/charsets/tryit.asp?deci=9728) &#9728; `&#9728;`
[Moon Icon](https://www.w3schools.com/charsets/tryit.asp?deci=9790) &#9790; `&#9790;`

Reply:
Yes

Reply:
> Yes

completed